crates/tags/src/tag.rs

```diff
@@ -1,21 +1,27 @@
 // SPDX-FileCopyrightText: 2022 Profian Inc. <opensource@profian.com>
 // SPDX-License-Identifier: Apache-2.0
 
+use io::Error;
 use std::hash::Hash;
+use std::io;
 use std::ops::{Deref, DerefMut};
 use std::str::FromStr;
 
 use drawbridge_http::http::{Request, StatusCode};
 use drawbridge_http::{async_trait, FromRequest};
 
-use semver::{Error, Version};
 use serde::{Deserialize, Serialize};
 
 #[derive(Debug, Clone, PartialEq, Eq, Hash, Serialize, Deserialize)]
-pub struct Tag(Version);
+pub struct Tag(String);
+
+impl Tag {
+    pub const TYPE_ENTRY: &'static str = "application/vnd.drawbridge.tag.v1+json";
+    pub const TYPE_JOSE: &'static str = "application/jose+json";
+}
```

Comment:
These don't seem to be used, also why a separate MIME type for tags, rather than using the entry type?